Users/Glob_Patterns.rst

```diff
@@ -174,14 +209,10 @@ brackets. Parentheses that have no match are ignored as well as
 
```

Comment:
I think the heading there displays pattern the same way it should be used. I don't think it's about highlighting thing. (it's fine `` highlights the content, and it has no trouble being used with \)  but coming to usage of glob pattern, while using in command line we should provide \ before each *.